test/phoenix_component_test.exs

```diff
@@ -3,6 +3,8 @@ defmodule Phoenix.ComponentTest do
 
   use Phoenix.Component
 
+  def test_single_arg_with_defaults(js \\ %Phoenix.LiveView.JS{}), do: js
```

Comment:
We have a test at the bottom called "keeps Elixir semantics". Let's add this around that. :)